src/app/backend/apihandler.go

```diff
@@ -90,6 +101,22 @@ func (apiHandler *ApiHandler) handleDeploy(request *restful.Request, response *r
 	response.WriteHeaderAndEntity(http.StatusCreated, cfg)
 }
 
+// Handles namespace creation API call.
+func (apiHandler *ApiHandler) handleNamespaceCreation(request *restful.Request,
+	response *restful.Response) {
+	cfg := new(NamespaceSpec)
```

Comment:
This variable should not be named `cfg`, it does not describe what it really is. Maybe `namespaceSpec`? I see that line 91 has the same issue (probably a leftover). Can you fix both?